deno.jsonc

```diff
@@ -0,0 +1,666 @@
+// auto-generated by package/src/common/create-deno-config.ts
+// see dev-docs/update-deno_jsonc.md
+{
+  "lint": {
+    "files": {
+      "include": [
+        "src/"
+      ],
+      "exclude": [
+        "src/resources/create/extensions/revealjs-plugin/_extensions/qstart-filesafename-qend/qstart-filesafename-qend.js",
+        "src/resources/editor/",
+        "src/core/lib/external/",
+        "src/vendor/",
+        "src/resources/vendor/",
+        "src/resources/formats/pdf/pdfjs/",
+        "src/resources/deno_std/",
+        "/Users/cscheid/repos/github/quarto-dev/quarto-cli/package/pkg-working/share/extensions/quarto/video/resources/videojs/video.min.js",
```

Comment:
@cscheid Is the absolute really paths expected ?

Reply:
No, they should 100% be relative.

Reply:
Fixed in 57760013b880406ea1821d3428e5e2c70c236011 

thanks